src/global_context.rs

```diff
@@ -39,6 +41,8 @@ pub struct CommandLine {
     pub basic_telemetry: bool,
     #[structopt(long, short="s", help="Send snippet telemetry (code snippets)")]
     pub snippet_telemetry: bool,
+    #[structopt(long, default_value="127.0.0.1", help="Binds by default to 127.0.0.1. This is compatible with having an HTTP server at the same time.")]
+    pub lsp_host: String,
```

Comment:
Hmm it's fine, but I mildly prefer to have just 1 parameter for host, just to reduce clutter (in code and in --help)